packages/trip-form/src/styled.ts

```diff
@@ -21,31 +28,28 @@ export const FloatingSettingLabel = styled(SettingLabel)`
   float: left;
 `;
 
-export const DateTimeSelector: DateTimeSelectorAndSubComponents = styled.div``;
-
-DateTimeSelector.DepartureRow = styled.div`
-  box-sizing: border-box;
-  > * {
-    box-sizing: border-box;
-    width: 33.333333%;
-    padding: 0px 5px;
-  }
-`;
+export const DateTimeSelector: DateTimeSelectorAndSubComponents = styled.div`
+  border-radius: 5px;
+  background: ${grey[50]};
```

Comment:
Why are we not using the base color and text color from the variables? I think it's possible to set a fallback too

Reply:
I forgot we talked about this! Added